### PR TITLE
generate API client

### PR DESCRIPTION
**Pull request by printminion-co**
in order to re-generate API client

```shell
./update_api.sh api.example.lan:10443
```

### CODE REVIEW

File: .github/workflows/block-unconventional-commits.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+concurrency:
+  group: block-unconventional-commits-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  block-unconventional-commits:
+    name: Block unconventional commits
+
+    runs-on: ubuntu-latest-low
```
Comment (bromiesTM) [inline .github/workflows/block-unconventional-commits.yml]: using `ubuntu-latets-low` makes it very unlikely that the action runs at all, as they usually just time out

Comment (bromiesTM) [reply .github/workflows/block-unconventional-commits.yml]: nvm

File: README.md [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+```
+
+## API Endpoints
+
+All URIs are relative to *https://API_HOST/easynextcloud-mail-notification*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*EventAPIApi* | [**processRemoteWipeFinishedEvent**](docs/Api/EventAPIApi.md#processremotewipefinishedevent) | **POST** /event/{brand}/remote-wipe-finished | Nextlcoud user gets informed that a remote wipe for a specific device has finished
+*EventAPIApi* | [**processRemoteWipeStartedEvent**](docs/Api/EventAPIApi.md#processremotewipestartedevent) | **POST** /event/{brand}/remote-wipe-started | Nextlcoud user started a remote wipe for a specific device
```
Comment (bromiesTM) [inline README.md:77]: typo:
Mom: We have a Cloud Storage at home
Cloud Storage at home: `Nextlcoud`

Comment (printminion-co) [reply README.md:77]: Autogenerated via openapi definition.. ..we can only communicate it to  api developers #sad

**Pull request closed without merging**